cpp/cmake/IceConfig.cmake

```diff
@@ -24,10 +27,10 @@ endif()
 if(Ice_SOURCE_BUILD)
   list(APPEND ice_include_path_suffixes "cpp/include")
   list(APPEND ice_bin_path_suffixes "cpp/bin")
-  list(APPEND ice_lib_path_suffixes_release "cpp/lib")
+  list(APPEND ice_lib_path_suffixes "cpp/lib")
 
   if (Ice_ARCHITECTURE)
```

Comment:
What is Ice_ARCHITECTURE and when is it not-set?

Reply:
It's computed from a few values (see above).

- If `CMAKE_LIBRARY_ARCHITECTURE` is set we use it. For example this is set on Ubuntu to something like "x86_64-linux-gnu"
- Otherwise if not set and we're on Windows we'll set it to either x64 or Win32

If it's not set nothing happens, it's a matter of checking `lib` vs `lib/<arch>`